### PR TITLE
SME, QE, and peer review PR for Bxmsdoc 3352 master Rework the GSG for DS using the 7.0 traffic version scenario

### PR DESCRIPTION
**Pull request by michelehaglund**
Original Jira: https://issues.jboss.org/browse/BXMSDOC-3352
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3352/

Recreated the 7.1 GSG for DS by reverting back to the Traffic violation scenario and testing same with the 7.1 build.

### CODE REVIEW

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[id='guided-suspension-rule-conditions-proc']
+= Setting the Suspension rule conditions
+
+Set the `Suspension` rule conditions that are used to determine the driver's violation.
+
+.Prerequisite
+You have created the Driver License Suspension rule.
+
+.Procedure
+. Click (image:getting-started/6176.png[]) next to the *WHEN* label to open the *Add a condition to the rule* window. . . Select *Driver* and click *Ok*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: Check the formatting. Select Driver needs it's own line.

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
++
+
+.Create new Guided Rule window
+image::getting-started/add-condition.png[]
+. Click the *There is a Driver* label to open the *Modify constraints for Driver* window.
+. Enter `driver` in the *Variable name* field and click *Set*.
+. Click the *There is a Driver[driver]* and click *Expression editor*.
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `previousPts` and click *Set*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: The : looks a little weird.

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+. Click *There is a number* label to open the *Modify contraints for Driver* window.
+. Click *Expression editor* and select *intValue()* from the
+*[not bound]:* *Choose* menu.
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `totalNewPoints` and click *Set*.
+. Click *click to add pattern* and select *Violation* from the *choose fact type* pull-down menu.
+. Click *All Violation with:* to open the *Modify contraints for Violation* window and select *points* from the *Add a restriction on a field* pull-down menu.
+. Click *please choose* next to the *points* label and select *greater than*.
+. Click (image:getting-started/6191.png[]), then click *Literal value*.
+. Click the *points* label to open the *Add a field* window and enter `vPoints` and click *Set*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: I'd make this a series and eliminate one and

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc (hunk 1 above)
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: Formatting AB4

File: doc-content/enterprise-only/getting-started/business-rules-create-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[id='guided-rule-licence-suspension-create-proc']
+= Creating the Driver License Suspension rule
+Create the `Driver license suspension` rule using the Guided Rule wizard.
+
+.Prerequisite
+
+Created both the `Violation` and `Driver` data objects.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/business-rules-create-proc.adoc]: You have created both....

File: doc-content/enterprise-only/getting-started/decision-table-create-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+= Creating a traffic violation guided decision table
+
+Use the Guided Decision Table designer to create the traffic violation guided decision table, which specifies the driver's specific violation and the resulting fine and points.
+
+.Prerequisite
+
+Created both the `Violation` and `Driver` data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver_department_traffic_violations*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-create-proc.adoc]: Should be two steps.

File: doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,26 @@
+[id='penalty-columns-proc']
+= Inserting `Fine Amount` and `Points` columns
+
+The `Fine Amount` and `Points` columns contain the fines and points based on the corresponding `Violation Type` field values.
+
+.Prerequisite
+
+Inserted the `Violation Type` column.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc]: You have inserted. I actually think this doesn't give enough context. Maybe something like 'You have inserted the Violation Type column in (where ever), in Decision Central. The idea is the prerequisites should give enough for the reader get to the starting point for the first step, assuming they haven't read the rest of the doc (but would know how to get to this point). Might be worth reviewing all of your prereqs from this POV.

File: doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+. Select *Field* and then *fineAmount* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Fine Amount` in the *Header (description)* field.
+. Select the *Update engine with changes* option and click *Finish*.
++
+
+.Fine Amount header
+image::getting-started/fine-amount.png[]
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern*, and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Field* and then *points* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Points` in the *Header (description)* field.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc]: ....options, and then (or just and) select Additional info, and then (add comma after info)

File: doc-content/enterprise-only/getting-started/decision-table-rows-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+After you have created your columns in the guided decision table, you can add rows and define rules within the decision table designer.
+
+.Prerequisite
+
+You have created the `Violation Type`, `Fine Amount`, and `Points` columns.
+
+.Procedure
+. Click *Editor* -> *Insert* -> *Append row*. Repeat this step to add a total of five table rows.
+. Fill out the table as shown in the following example.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-rows-proc.adoc]: ...example:

File: doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,62 @@
+[id='violation-columns-proc']
+= Inserting Violation Type columns
+
+The `Violation Type` column contains the violation details such as the the driver's speed and if the driver was under the influence of drugs or alcohol.
+
+.Prerequisite
+
+Created the traffic violation guided decision table.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc]: You have created...

File: doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+.Add a new column window
+image::getting-started/new-column.png[]
+. Select *Ruleflow-group* and click *Finish*.
+. Expand *Attribute columns* and enter `trafficViolation` in the *Default value* field.
++
+
+.Attribute columns window
+image::getting-started/attrib-column.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* -> *+Create a new Fact Pattern*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc]: ....Column and then select...

File: doc-content/enterprise-only/getting-started/new-project-ds-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
+[id='decision-services-project-create-proc']
+= Creating the traffic violations project
+A project is the container for assets, such as data objects, guided decision tables, and guided rules. For this tutorial, you will create the *Driver_department_traffic_violations* project.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/new-project-ds-proc.adoc]: No comma after assets (assets such as...)

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,49 @@
+[id='testing-speed-limit-proc']
+= Testing the speed limit scenario
+
+Test the speed limit scenario using the data that you specified when you created the speed violation guided decision table.
+
+.Prerequisites
+
+* Created the `Driver_department_traffic_violations` project
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: You have ... and period at the end of each

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+.. *Package*: select `com.myspace.driver_department_traffic_violations`.
+. Click *Ok*.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Violation* from the *Insert a new fact* pull-down menu.
+. Enter `violation` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *Insert 'Violation'[violation]* to open the *Choose a field to add* window.
+. Select *speedLimit* from the *Choose a field to add* pull-down menu and click *OK*.
+. Click (image:getting-started/6191.png[]), then click *Literal value* next to *speedLimit*.
+. Click *Literal value* in the *Field value* window, then enter `40` in the *speedLimit* field.
+. Click *Insert 'Violation'[violation]* and select *type* from the *Choose a field to add* pull-down menu in the *Choose a field to add* window, and click *OK*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: Make a series and lose an and

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+. Select *points* from the *Choose a field to add* pull-down menu and click *OK*.
+. Enter `1` in the *points: equals* field.
+. Click *Save*, then click *Save* to confirm your changes.
+. Click *Run scenario*.
++
+
+.Speed test results screen
+image::getting-started/speedtest_results.png[Speed test results screen]
+
+The Reporting section at the bottom of the window should show a Success message. This is the expected result as the values and conditions set in the test scenario meet the requirements as specified in speed violation guided decision table.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: I would rephrase to lose 'should'. Something like:
If the values and conditions set in the test scenario meet the requirements as specified in speed violation guided decision table, the Reporting section at the bottom of the window displays a Success message.

File: doc-content/enterprise-only/getting-started/test-speed-scenarios-con.adoc [hunk 15]
```diff
@@ -0,0 +1,8 @@
+[id='test-scenarios-con']
+= Test Scenarios
+
+Test Scenarios in {PRODUCT} enable you to validate the functionality of rules, models, and events before deploying them into production. A test scenario uses data for conditions that resemble an instance of your fact or project model. This data is matched against a given set of rules and if the expected results match the actual results, the test is successful. If the expected results do not match the actual results, then the test fails.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-speed-scenarios-con.adoc]: replace given with specific or delete. IBM SG: given adj
Do not use to mean “particular” or “specific.” Consider whether any adjective is necessary.

File: doc-content/enterprise-only/getting-started/test-suspend-proc.adoc [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+. Click *+Expect* to open the *New expectation* window and click *Add* next to *Fact value: driver*.
+. Click *Driver 'driver' has values:* to open the *Choose a field to add* window.
+. Select *state* from the *Choose a field to add* pull-down menu and click *OK*.
+. Enter `suspend` in the *state: equals* field.
+. Click *Save*, then click *Save* to confirm your changes.
+. Click *Run scenario*.
++
+
+.Result
+The rule is fired and the driver's license is suspended because the total number of points is >= 20.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-suspend-proc.adoc]: Why not spell out greater than instead of the symbols?

File: doc-content/enterprise-only/getting-started/test-suspend-proc.adoc [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
++
+
+.Suspension test results screen
+image::getting-started/suspendtest_results.png[Suspension test results screen]
+
+The Reporting section at the bottom of the window should show a Success message. This is the expected result as the values and conditions set in the test scenario meet the requirements as specified when you set the Driver License Suspension rules and actions.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-suspend-proc.adoc]: AB4

File: doc-content/enterprise-only/getting-started/test-violation-based-proc.adoc [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+. Click (image:getting-started/6191.png[]) next to *points*, click *Literal value*, then enter `5` in the *points* -> *violation2* field.
+. In the *totalPoints* field, change the value from `10` to `5`.
+. Click *Save*, then click *Save* to confirm your changes.
+. Click *Run scenario*.
++
+
+.Suspension test results screen
+image::getting-started/suspend_multi_test_results.png[Suspension test results screen]
+
+The Reporting section at the bottom of the window should show a Success message. This is the expected result as the values and conditions set in the test scenario meet the requirements as specified when you set the Driver License Suspension rules and actions.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/test-violation-based-proc.adoc]: AB4

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataModeller/sections/FirstSteps-section.adoc [hunk 19]
```diff
@@ -14,7 +14,7 @@ To start the creation of a data model inside a project, take the following steps
 image::Workbench/Authoring/DataModeller/authoring.jpg[align="center"]
 
 +
-. Open the Data Modeller tool by clicking on a Data Object file, or using the "Create New Asset -> Data Object" menu option. Set Data Object name to "PurchaseOrder" and click Ok.
+. Open the Data Modeller tool by clicking on a Data Object file, or using the "Add Asset -> Data Object" menu option. Set Data Object name to "PurchaseOrder" and click Ok.
```
Comment (emmurphy1) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataModeller/sections/FirstSteps-section.adoc]: by clicking, not clicking on. Delete comma before or. This is two steps. Bold OK.

File: doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc [hunk 20]
```diff
@@ -0,0 +1,30 @@
+[id='guided-suspension-rule-actions-proc']
+= Setting the Suspension rule actions
+
+Set the `Suspension` rule actions that are used to determine a driver's resulting penalties, including points and fine amounts, based on the `Suspension` rule conditions.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc]: Fixed instances of coded Suspension and Driver instances. Good catch :)

File: doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+
+.Prerequisite
+You have created set the Suspension rule conditions
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc]: Fixed all instances in prereqs.

File: doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+
+.Procedure
+. Click *(show options…​)*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc]: Stet. The procedure flow is correct in this instance.

File: doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc [hunk 23]
```diff
@@ -0,0 +1,30 @@
...
++
+
+.show options selection
+image::getting-started/show_options.png[]
+. Click (image:getting-started/6176.png[]) next to the *THEN* label and select *Change field values of driver*, then click *Ok*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-actions-proc.adoc]: Corrected.

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: Good catch. Fixed.

File: doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc (hunk 3 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/business-rules-create-proc.adoc (hunk 4 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/business-rules-create-proc.adoc]: Corrected throughout.

File: doc-content/enterprise-only/getting-started/decision-table-create-proc.adoc (hunk 5 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-create-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc (hunk 6 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc]: Corrected. Thank you.

File: doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc (hunk 7 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc]: Done.

File: doc-content/enterprise-only/getting-started/decision-table-rows-proc.adoc (hunk 8 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-rows-proc.adoc]: Done.

File: doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc (hunk 9 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc]: Done throughout doc

File: doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc (hunk 10 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/decision-table-violation-columns-proc.adoc]: Done.

File: doc-content/enterprise-only/getting-started/new-project-ds-proc.adoc (hunk 11 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/new-project-ds-proc.adoc]: Corrected.

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc (hunk 12 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: Corrected throughout.

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc (hunk 13 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/test-speed-proc.adoc (hunk 14 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-speed-proc.adoc]: Fixed same for all test three test scenario topics.

File: doc-content/enterprise-only/getting-started/test-speed-scenarios-con.adoc (hunk 15 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-speed-scenarios-con.adoc]: changed to "specific"

File: doc-content/enterprise-only/getting-started/test-suspend-proc.adoc (hunk 16 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-suspend-proc.adoc]: Cause I don't want to :)

File: doc-content/enterprise-only/getting-started/test-suspend-proc.adoc (hunk 17 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-suspend-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/test-violation-based-proc.adoc (hunk 18 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/test-violation-based-proc.adoc]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataModeller/sections/FirstSteps-section.adoc (hunk 19 above)
Comment (michelehaglund) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataModeller/sections/FirstSteps-section.adoc]: ah.. These last topics are not mine.
